websites/O/OtakuEyes/presence.ts

```diff
@@ -0,0 +1,47 @@
+const presence = new Presence({
+    clientId: "784067980086607952",
+  }),
+  startTimestamp: number = Math.floor(Date.now() / 1000);
+
+async function getStrings() {
+  return presence.getStrings({
+    browsing: "general.browsing",
+  });
+}
+let strings: Awaited<ReturnType<typeof getStrings>>;
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "https://i.imgur.com/wFmlFjg.png",
+    startTimestamp,
+  };
+  const pathnameArray = document.location.pathname.split("/");
+  const page = pathnameArray[1];
+  const episode = pathnameArray[3];
+  const [showCover] = await Promise.all([
+    presence.getSetting < boolean > "cover",
```

Comment:
You probably missed some parentheses.
```suggestion
    presence.getSetting<boolean>("cover"),
```